Doc/library/types.rst

```diff
@@ -311,6 +311,25 @@ Standard names are defined for the following types:
    .. versionchanged:: 3.9.2
       This type can now be subclassed.
 
+   .. attribute:: __origin__
```

Comment:
This section is a duplicate of https://docs.python.org/3/library/stdtypes.html#special-attributes-of-genericalias-objects. Maybe we should just link to that instead?

Reply:
Oh thanks, I missed that! I'll add a section there instead.

Reply:
Fixed in 81cbc83ebaa450d2a7232281a7239cbe969d5566